docs/config.md

```diff
@@ -776,6 +776,8 @@ GnuPG or SSH signing keys.
 
 To do this you need to configure a signing backend.
 
+Setting the backend to `""` (empty string) disables signing.
```

Comment:
nit: might be better to use `"none"`. `signing.backend` is enum-like, and is similar to `core.fsmonitor`.

Reply:
Makes sense. Done.